src/components/MaOption/MaOptionCheckbox.css

```
@@ -16,9 +16,23 @@
 .ma-option--checkbox .description {
   /* .indicator is absolute-positioned. We need to make room for the text */
   padding-left: calc(var(--indicator-size) + 0.5rem);
-  margin-top: 2px;
   color: var(--color-gray-dark);
 }
+.ma-selector-xsmall {
+  &.ma-option--checkbox .description {
+    margin-top: 4px;
+  }
+}
+.ma-selector-small {
+  &.ma-option--checkbox .description {
+    margin-top: 3px;
+  }
+}
+.ma-selector-medium {
+  &.ma-option--checkbox .description {
+    margin-top: 2px;
+  }
+}
```

this nesting is useless no? And `ma-selector-xxx` is always in `ma-option--checkbox` so why not:
```suggestion
.ma-selector-xsmall .description {
  margin-top: 4px;
}
.ma-selector-small .description {
  margin-top: 3px;
}
.ma-selector-medium  .description {
  margin-top: 2px;
}
```

---

nesting removed!